code/game/objects/items/weapons/grenades/smokebomb.dm

```diff
@@ -1,13 +1,18 @@
 /obj/item/grenade/smokebomb
-	desc = "It is set to detonate in 2 seconds."
 	name = "smoke bomb"
+	desc = "A grenade filled with chemical agents that will turn into a dense smoke when detonated, making it impossible to see through without specialised optics."
 	icon = 'icons/obj/grenade.dmi'
 	icon_state = "smoke"
-	det_time = 20
+	det_time = 2 SECONDS
+	display_timer = FALSE
```

Comment:
This should be `TRUE`, no?

Reply:
Smoke bombs cannot be modified on live.

Time was stated on `examine()` but I've added an extra variable to handle it so it's not snowflakey. Tested that everything still displays correctly.

Reply:
I know. But your code was missing a `!` (which you added in the latest commit) which makes it now work properly.

![image](https://github.com/user-attachments/assets/3631e1f5-9ddc-480d-a10c-4103cee9edda)

The extra variable makes it a lot more readable and clear though, very good 👍